fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1766,6 +1766,12 @@ public class Config extends ConfigBase {
     @ConfField(mutable = false, expType = ExperimentalType.EXPERIMENTAL)
     public static boolean enable_fqdn_mode = false;
 
+    /**
+     * enable use odbc table
+     */
+    @ConfField(mutable = true, masterOnly = true)
+    public static boolean enable_odbc_table = false;
```

Comment:
use `description` field in `ConfField` annotation to add document of this config.